qiskit/pulse/commands/parametric_pulses.py

```diff
@@ -17,8 +17,8 @@
 
 # pylint: disable=unused-import
 
-from qiskit.pulse.pulse_lib import (ParametricPulse, Gaussian, GaussianSquare,
-                                    Drag, Constant, ConstantPulse)
+from qiskit.pulse.library import ParametricPulse, Gaussian, GaussianSquare, Drag, Constant, ConstantPulse
+
```

Comment:
```suggestion
from qiskit.pulse.pulse_lib import (ParametricPulse, Gaussian, GaussianSquare,
                                    Drag, Constant, ConstantPulse)
```
This is one of the lines causing a style error, because the line length is greater than 100. Since this file is deprecated anyway (all of `commands/` is going to be removed), I don't think we have to worry about updating the import path.